test/TestUtils.py

```diff
@@ -233,3 +233,24 @@ def test_logger_debug(caplog):
     )
 
     assert expected_info in caplog.record_tuples
+
+
+def test_cli_auto_detect(capfd):
+    """Test that run without arguments it will detect and lint the entire repository."""
+    result = os.system("ansible-lint -v -p --nocolor")
```

Comment:
Let's use a bit more high-level subprocess API. Also I prefer to be explicit about the interpreter in order to not hit some other installation of ansible-lint accidentally:
```suggestion
    cmd = sys.executable, "-m", "ansible-lint", "-v", "-p", "--nocolor"
    result = subprocess.run(cmd).returncode
```